adapters/repos/db/crud_integration_test.go

```diff
@@ -1942,7 +1941,16 @@ func TestCRUDWithEmptyArrays(t *testing.T) {
 			Class: classNameWithRefs,
 			Properties: map[string]interface{}{
 				"stringProp": "some prop",
-				"refProp":    models.MultipleRef{},
+				// due to the fix introduced in https://github.com/semi-technologies/weaviate/pull/2285,
+				// MultipleRef's will appear as empty []string when no actual refs are provided for an
+				// object's reference property.
+				//
+				// when this obj1 is unmarshalled from storage, refProp will be represented as []string,
+				// because it is an empty reference property. so when comparing obj1 with the result of
+				// repo.Object, we need this refProp here to be a []string. Note that this is due to our
+				// usage of storobj.Object.MarshallerVersion 1, and future MarshallerVersions may not
+				// have this ambiguous property type limitation.
+				"refProp": []string{},
```

Comment:
hmm, then I'm thinking of maybe we should change this type to `[]interface{}` not `[]string{}` wdyt?

Reply:
I don't see an issue with that. It generalizes things which maybe makes it more clear that at this point, we don't know what the type is. Whereas with `[]string`, it could give the impression that we are expecting that specific type.